ipv8/REST/dht_endpoint.py

```diff
@@ -388,7 +388,7 @@ def should_sign(self, block):
                 # Try to add the current chunk to the DHT; if it works, move to the next, otherwise retry
                 try:
                     await self.dht.store_value(self._hashed_dht_key, blob_chunk[0])
-                except:
+                except (RuntimeError, PackError):
```

Comment:
The `store_value` function throws a `DHTError` instead of a `RuntimeError`.

Reply:
Ah, oops. Will fix 👍

Reply:
Fixed